### DISCUSSION TITLE
@loopback/example-log-extension not write any log?

### DISCUSSION THREAD

**Discussion by Vn-ChemGio** [loopbackio/loopback-next#7763] [Q&A]
Hi guy.
I'm using loopback 4 and try to use module @loopback/example-log-extension to display log

In application.ts,  I were wrap BootMixin by LogMixin
```
BootMixin(
  ServiceMixin(LogMixin(RepositoryMixin(RestApplication)),
))
```
LogMixin(BootMixin(
  ServiceMixin(RepositoryMixin(RestApplication)),
))

And add @log(LOG_LEVEL.DEBUG) to controller 

export class PingController {
  constructor(@inject(RestBindings.Http.REQUEST) private req: Request) {}

```
  // Map to `GET /ping`
  @log(LOG_LEVEL.DEBUG)
  @get('/ping')
  @response(200, PING_RESPONSE)
  ping(): object {
    // Reply with a greeting, the current time, the url, and request headers
    return {
      greeting: 'Hello from LoopBack',
      date: new Date(),
      url: this.req.url,
      headers: Object.assign({}, this.req.headers),
    };
  }
}
```
But when I call  api localhost:3000/ping 
Nothing show in console log
=======================
So, what I have to do? or where I can see log file

**Comment by achrinza** [comment]
It looks like you're trying to use one of the example packages (`@loopback/example-*`). These are meant to be used more as example code of actual LoopBack 4 apps rather than extensions.

Perhaps [@loopback/logging](https://github.com/loopbackio/loopback-next/tree/master/extensions/logging) is what you were looking for?

**Reply by Vn-ChemGio** [reply]
Thank @achrinza  for your answear.
I were try @loopback/example-log to using example package to test,  => If  it working, I will implement to my project
But nothing to see in console.log.

When I read document of @loopback/logging,  I'm not sure my system is supporting FLUENTD_SERVICE , because my devops team told me we using ES log
But this time I have to write log in console.log.
So @achrinza how we work with @loopback/example-log or @loopback/logging to write log in console.log
